### DISCUSSION TITLE
request image build excludes file

### DISCUSSION THREAD

**Discussion by brcisna** [ltsp/ltsp#249] [Q&A]
Hello All,

trying out the new generation LTSP for first time,after having used LTSP for about 15 years now.

Would it be possible to include an 'excludes.file' for building the client image such as was in the LTPS5 branch?
Example: Have many many media files amounting to many gigs that the client really don't need to access.
This would simply speed up the client image build time especially on an older server machine.

Thank You

**Comment by alkisg** [comment]
Hi, it's the first ltsp.conf parameter: https://ltsp.org/man/ltsp.conf/

**Comment by brcisna** [comment]
Hi Alkis ,

thank you for this information.

Added a line to bottom of newly generated ltsp.conf file,
ADD_IMAGE_EXCLUDES="/etc/ltsp/add-image.excludes"

I added a file in the /etc/ltsp directory add-image.excludes  (perms are 0644)

In this file i added three directories listings 
dir1/*
dir2/*
dir3/*

But the built client image is still the same file size.

I'm sure i am missing something obvious.

I like the simplicity of the new LTSP building platform BTW!

Thank You.

**Comment by alkisg** [comment]
Hi, if you added it to the bottom, then it probably went under [clients], while you need to put it under the [server] section.

**Comment by brcisna** [comment]
Hi Alkis,

You were spot on. I missed the [sections] declarations in this file.
After moving the comment up under the [server] component on the file, and doing an,,  ltsp image /
the squashfs   client file size is way down in size.

Thank You
